x-pack/plugins/apm/public/services/rest/apm_observability_overview_fetchers.ts

```diff
@@ -24,6 +25,7 @@ export const fetchObservabilityOverviewPageData = async ({
         start: new Date(absoluteTime.start).toISOString(),
         end: new Date(absoluteTime.end).toISOString(),
         bucketSize,
+        intervalString,
```

Comment:
I think `bucketSize` and `intervalString` both stores the same interval in seconds, only difference being that `bucketSize` is a number (`5`) and `intervalString` is a string (`5s`).

Would be great if the names reflected this, a couple of suggestions:
 - `bucketSize` / `bucketSizeString`
 - `interval` / `intervalString`
 - `intervalNumber`/ `intervalString`

Another approach would be an object like:
`bucketSize.asNumber()` and `bucketSize.asString()`